src/main/kotlin/no/nav/yrkesskade/skadeforklaring/model/Helseinstitusjon.kt

```diff
@@ -1,10 +1,12 @@
 package no.nav.yrkesskade.skadeforklaring.model
 
 import io.swagger.v3.oas.annotations.media.Schema
+import javax.validation.constraints.Max
 
 data class Helseinstitusjon(
     @Schema(example = "ja", description = "Beskriver om skadelidt har vært hos medisinsk behandler. Gyldige verdier er 'ja' eller 'nei'")
     val erHelsepersonellOppsokt: String,
+    @Max(150)
```

Comment:
Mener `@Max` er for tall, mens for strenger så skal man bruke `@Size(max = ...)`

Reply:
Bra sett! Jeg brukte Size først, men så tenkte jeg at Max gjorde samme jobben, men det gjør den kanskje ikke